common/app-starters-task-launch-request-common/src/main/java/org/springframework/cloud/stream/app/tasklaunchrequest/DataFlowTaskLaunchRequestAutoConfiguration.java

```diff
@@ -42,25 +45,21 @@
 @EnableConfigurationProperties(DataflowTaskLaunchRequestProperties.class)
 public class DataFlowTaskLaunchRequestAutoConfiguration {
 
-	private static final Log log = LogFactory.getLog(TaskLaunchRequestTransformer.class);
+	private static final Log log = LogFactory.getLog(TaskLaunchRequestFunction.class);
```

Comment:
Why do you use different class for logging in this `DataFlowTaskLaunchRequestAutoConfiguration`?
What is the point to mislead?